README.md

```diff
@@ -1,3 +1,96 @@
+[![Build Status](https://travis-ci.org/acquia/headless-lightning.svg?branch=master)](https://travis-ci.org/acquia/lightning)
+
 # Headless Lightning
+Headless Lightning is a more opinionated version of the [Lightning Drupal](https://github.com/acquia/lightning)
+distribution as a backend for decoupled applications.
+
+## Quick Start
+To build the codebase:
+
+    composer create-project acquia/lightning-project:dev-headless --no-interaction
+
+This will create a new directory, `lightning-project` which contains a `docroot`
+folder. This is where you should point your web server.
+
+Just like any Drupal project, you will need a working [LAMP stack](https://www.drupal.org/node/2461571)
```

Comment:
We should reword this, since you don't strictly need a LAMP stack. This is Drupal, and it will run on any environment that Drupal runs on. Can this be rephrased like "You'll need an environment in which Drupal can be run" (or similar) and link to the relevant documentation?